src/main/java/PetShelterApp.java

```diff
@@ -0,0 +1,62 @@
+
+import org.petshelter.model.Pet;
+import org.petshelter.model.Shelter;
+import org.petshelter.service.PetShelterService;
+import org.petshelter.service.PetShelterServiceImpl;
+import org.petshelter.dataservice.ShelterDataService;
+import java.util.Scanner;
+
+import static java.lang.System.out;
+
+public class PetShelterApp {
+    private static final Scanner scanner = new Scanner(System.in);
+    private static final Shelter shelter = new Shelter();
+    private static final PetShelterService petShelterService = new PetShelterServiceImpl(shelter);
+
+    public static void main(String[] args) {
+        ShelterDataService.loadShelterData(shelter);
+
+        while (true) {
+            displayMenu();
+            int choice = scanner.nextInt();
+            scanner.nextLine();
+
+            switch (choice) {
+                case 1:
+                    out.println("Enter pet details:");
+                    out.print("Name: ");
+                    String name = scanner.nextLine();
+                    out.print("Breed: ");
+                    String breed = scanner.nextLine();
+                    out.print("Age: ");
+                    int age = scanner.nextInt();
+                    Pet newPet = new Pet(name, breed, age);
+                    petShelterService.addPet(newPet);
```

Comment:
it's better to move createPet() logic into separate method for better readability